com.unity.shadergraph/Editor/Data/Nodes/NodeClassCache.cs

```diff
@@ -0,0 +1,112 @@
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Profiling;
+
+namespace UnityEditor.ShaderGraph
+{
+    [InitializeOnLoad]
+    internal static class NodeClassCache
+    {
+        private static Dictionary<Type,List<ContextFilterableAttribute>> m_KnownTypeLookupTable;
+
+        private static Dictionary<Type, List<ContextFilterableAttribute>> s_KnownTypeLookupTable
```

Comment:
This property isn't actually used. Since the static constructor will run anyway, you can safely remove it.

Reply:
ah wait thats my mistake, that was just an internal accessor to ensure lazy evals, I should be using that one in most cases

Reply:
though I suppose [InitializeOnLoad] constructor should make that impossible...I will remove